Templates/partials/run_header.jinja2

```diff
@@ -7,6 +7,10 @@
     <div class="run-header-content">
       <div class="run-title pt-4 pr-4 pb-4">
         <div class="title">
+          {% if run.has_archive_link %}
```

Comment:
This results in us running the full logic three times :/ Is there a way to store a variable when building the template?